extension/dnsmasq/dnsmasq.conf

```diff
@@ -46,6 +46,13 @@ log-facility=/dev/null
 
 strict-order
 
+# If you don't want dnsmasq to read /etc/hosts, uncomment the
+# following line.
+#no-hosts
+# or if you want it to read another file, as well as /etc/hosts, use
+# this.
+addn-hosts=/home/pi/.firewalla/run/dnsmasq_addn_hosts
```

Comment:
It's better to add this into FILTER_DIR of dnsmasq.js instead of changing static config file for compatibility of all platforms